extensions/telemetry/src/telemetry-preferences-store.ts

```diff
@@ -23,6 +23,10 @@ export class TelemetryPreferencesStore extends Store.ExtensionStore<TelemetryPre
     this.data.enabled = v
   }
 
+  protected fromStore(data: TelemetryPreferencesModel): void {
+    this.data = data
```

Comment:
Why is this needed?

Reply:
I changed `BaseStore` to be an `abstract class` with `fromStore` and `toJSON` as its two `abstract` methods. This was to force people writing new stores to remember that those are the two methods that **need** to be implemented. This was the only class that didn't, on the whole (now that you brought my mind back to it) this class should really be closer in design to the other stores and not keep `data` around at all, just a field `enabled`.